flink-runtime-web/src/test/resources/log4j-test.properties

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-log4j.rootLogger=OFF, console
+log4j.rootLogger=INFO, console
```

Comment:
revert

Reply:
Good catch. Sorry that I missed that. Will revert it.